home/rvagg/2021-02-exploration-report-small-filecoin-deals.md

```diff
@@ -54,7 +63,7 @@ Followed instructions at https://docs.filecoin.io/get-started/lotus/installation
 
  * Large AMD EPYC instance, Ubuntu 20.04
  * Installed dependencies (used Go Snap rather than Go tarball)
- * Cloned and build Lotus with CPU SHA support
+ * Cloned and build Lotus with CPU SHA support (learned later that these flags don't impact `lotus` compile, the SHA extnsions are only used for mining)
```

Comment:
Now that I read this I realize I mislead you *a little*. Yes, SHA acceleration has no effect on actual chainsync. But it does help noticeably when calculating the CommP to shove into the deal proposal ( underneath all layers it's still sha2-256 ). So a user making a ton of deals could potentially benefit, especially if they are not transfering the data ( offline deals of some capacity )